cpu/native/mtd/mtd_native.c

```diff
@@ -37,7 +37,7 @@ static int _init(mtd_dev_t *dev)
     FILE *f = real_fopen(_dev->fname, "r");
 
     if (!f) {
-        DEBUG("mtd_native: init: creating file %s\n", name);
+        DEBUG("mtd_native: init: creating file %s\n", _dev->fname);
```

Comment:
nice catch!

Reply:
The credit goes to @MichelRottleuthner (this comes from #7104) ;)

Reply:
ah, so this is kind of based on #7104? and we should merge that first?

Reply:
This is not really based on #7104, I simply copied this fix and the `vfs.h` part as well.
So we can merge independently, I guess only a rebase will be needed.